sample-app/src/main/java/com/uber/myapplication/MainActivity.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2017. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uber.myapplication;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+/** Sample activity. */
+public class MainActivity extends AppCompatActivity {
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
+    // Object x = null;
+    // x.hashCode();
```

Comment:
Why these commented lines?

Reply:
To test that NullAway was actually working.  If you uncomment them you get a compile error.

Reply:
I'll add a comment